ddl/ddl_api.go

```diff
@@ -4932,6 +4923,25 @@ func (d *ddl) AlterTableSetTiFlashReplica(ctx sessionctx.Context, ident ast.Iden
 	return errors.Trace(err)
 }
 
+func checkTiFlashSupportable(schema *model.DBInfo, tb table.Table) error {
+	// Ban alter tiflash related actions for tables in system database.
```

Comment:
This function is a check, not a ban, so that your comment should describe what kind of check it is checking, e.g.:

```go
// Memory table and system table does not support TiFlash
```